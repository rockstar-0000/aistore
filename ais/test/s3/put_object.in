echo "0123456789" > $OBJECT.txt
s3cmd --host=$HOST mb s3://$BUCKET $PARAMS --region us-west-1 --host-bucket="$HOST/%(bucket)"
ais bucket props set ais://$BUCKET checksum.type=md5
s3cmd --host=$HOST put $OBJECT.txt s3://$BUCKET/$OBJECT $PARAMS --region us-west-1 --host-bucket="$HOST/%(bucket)" // IGNORE
s3cmd --host=$HOST ls s3://$BUCKET $PARAMS --region us-west-1 --host-bucket="$HOST/%(bucket)" | wc -l
s3cmd --host=$HOST get s3://$BUCKET$OBJECT $OBJECT_copy.txt $PARAMS --region us-west-1 --host-bucket="$HOST/%(bucket)" // IGNORE
cat $OBJECT_copy.txt
rm $OBJECT.txt // IGNORE
rm $OBJECT_copy.txt // IGNORE
s3cmd --host=$HOST rm s3://$BUCKET$OBJECT $PARAMS --region us-west-1 --host-bucket="$HOST/%(bucket)"  // IGNORE
s3cmd --host=$HOST rb s3://$BUCKET $PARAMS --region us-west-1 --host-bucket="$HOST/%(bucket)"
