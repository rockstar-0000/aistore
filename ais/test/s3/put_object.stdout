Bucket 's3://$BUCKET/' created
"checksum.type" set to:"md5" (was:"xxhash")

Bucket props successfully updated.
1
0123456789
Bucket 's3://$BUCKET/' removed
